doc/source/examples/dem/plate-discharge/plate-discharge.rst

```diff
@@ -0,0 +1,496 @@
+
+
+Post-Processing Code
+~~~~~~~~~~~~~~~~~~~~
+
+The data is extracted with the Lethe PyVista tool and post-processed with custom functions in the files ``pyvista_utilities.py`` and ``log_utilities.py``.
+Extraction, post-processing and plotting are automated in the script ``plate-discharge_post-processing.py``:
+
+.. code-block:: text
+  :class: copy-button
+
+  python3 plate-discharge_post-processing.py
+
+The script will generate the figures. If you want to modify the path or the filenames, you have to modify the script.
+
+Performance Analysis
+~~~~~~~~~~~~~~~~~~~~
+
+The log files (outputs displayed in the terminal) are read to extract the simulation and wall times.
+
+The speedup is calculated with the baseline case as the reference. The results are plotted in the following figure, where the solid lines show the walltime during the simulation, the dashed lines show the speedup, and the points show to total speedup.
+
+
+.. figure:: images/performance.png
+   :alt: Performance results
+   :align: center
+   :name: plate-discharge-performance-graph
+
+   The walltime during the simulations (solid line) and the speedup (dashed line) for the performance enhancement methods with the Adaptive Sparse Contacts (ASC) and the Load Balancing (LB) compared to the baseline case.
+
+.. note::
+   The slight oscillations of the speedup are caused by the scientific notation format of the walltime by the timer feature after 1000 seconds. The walltimes are attenuated by the moving average, but the division operation for the speedup accentuates the lack of time precision.
+
+Without going into much details, the load balancing method will help the performance of the simulation from the start, since the particles move within the domain during the discharge. The load balancing allows to distribute the particles, therefore all their related computation, more evenly between the cores. Once the discharge of the particles is mostly done, and only a few particles are still falling from the top part, the performance gain brought by the load balancing stays constant since the load across the cores is already balanced.
+The adaptive sparse contacts method will help the performance of the simulation mostly when there are large areas of motionless particles. As it was showed in the video, those areas are located in the core of the pile at the top and at the corners of settled particles below the plate. This explains why the ASC gives a limited performance gain at the start of the simulation (only from the core of the pile) and an increasing gain through the simulation (accumulation of motionless particles at the bottom part). Given that both methods help the computation performance at different times, the combination of both methods gives the best performance as observed.
```

Comment:
```suggestion
Without going into much details, the load balancing method will help the performance of the simulation from the start, since the particles move within the domain during the discharge. The load balancing allows to distribute the particles, therefore all their related computation, more evenly between the cores. Once the discharge of the particles is mostly done, and only a few particles are still falling from the top part, the performance gain brought by the load balancing stays constant since the load across the cores is already balanced.

The ASC method will help the performance of the simulation mostly when there are large areas of motionless particles. As it was showed in the video, those areas are located in the core of the pile and at the top and corners of settled particles below the plate. This explains why the ASC gives a limited performance gain at the start of the simulation (only from the core of the pile) and an increasing gain through the simulation (accumulation of motionless particles at the bottom part). Given that both methods help the computation performance at different times, the combination of both methods gives the best performance as observed.
```